src/utils/getTokenMetadata.ts

```diff
@@ -5,10 +5,5 @@ import { rainbowTokenList } from '@/references';
 export default function getTokenMetadata(
   tokenAddress: string | undefined
 ): Omit<TokenMetadata, 'decimals' | 'chainId'> | undefined {
-  if (!tokenAddress) return undefined;
-  const metadata: TokenMetadata =
-    rainbowTokenList.RAINBOW_TOKEN_LIST[tokenAddress.toLowerCase()];
-
-  // delete chain  metadata
-  return omitFlatten(metadata, ['chainId', 'decimals']);
```

Comment:
can you help me understand how `decimals` which should be omitted here would impact the other flows?

Reply:
it wasn't being omitted, we've had a bunch of issues with this previously when we added BSC which you and I looked into and then had to pull bruno in